src/hooks/use-update-demo-site.tsx

```diff
@@ -96,6 +103,14 @@ export const DemoSiteUpdateProvider: React.FC< DemoSiteUpdateProviderProps > = (
 				} );
 				return response;
 			} catch ( error ) {
+				setErrorSites( ( prev ) => new Set( prev ).add( snapshot.atomicSiteId ) );
+				setTimeout( () => {
+					setErrorSites( ( prev ) => {
+						const next = new Set( prev );
+						next.delete( snapshot.atomicSiteId );
+						return next;
+					} );
+				}, UPDATED_MESSAGE_DURATION_MS );
```

Comment:
Let's not remove the error automatically. And let's clear the error when the user re-tries on line 37.


```suggestion
```